source/pdps-group-replication.rst

```diff
@@ -0,0 +1,233 @@
+
+The described solution **helps improve uptime**. It will help you to significantly reduce both RPO and RTO. Given the tightly coupled cluster solution approach, the failure of a single node will not result in service interruption. 
+
+Increasing the number of nodes will also improve the cluster resilience by the formula:
+
+.. code-block:: text
+
+   F = (N -1) / 2
+
+where:
+
+- ``F`` is the number of admissible failures
+- ``N`` is the number of nodes in the cluster.
+
+.. admonition:: Examples:
+ 
+  - In a cluster of 5 nodes, F = (5 - 1)/2 = 2. 
+    The cluster can support up to 2 failures. 
+
+  - In a cluster of 4 nodes, F = (4 - 1)/2 = 1. 
+    The cluster can support up to 1 failure. 
+
+This solution also allows for a more restrictive backup policy, dedicating a node to the backup cycle, which will help in keeping RPO low.
+
+As previously mentioned, disaster recovery is not covered by default by this solution. It will require an additional replication setup and controller. 
+
+Measurement and Monitoring
+--------------------------
+
+To ensure that database infrastructure is performing as intended or at its best, it is necessary to measure specific metrics and alert when some of these metrics are not in line with expectations. Periodic review of these measurements is also encouraged to promote stability and understand potential risks associated with the database infrastructure. 
+
+The following are the 3 aspects of database performance measurement and monitoring:
+
+* **Measurement** - To understand how a database infrastructure is performing, there is a need to measure multiple aspects of the infrastructure. With measurement it’s important to understand the impact of the sample sizes, sample timing and sample types.
+* **Metrics** - Metrics refer to the actual parts of the database infrastructure being measured. When we discuss metrics, more isn’t always better as it could introduce unintentional noise or make troubleshooting become overly burdensome.
+* **Alerting** - When one or many metrics of the database infrastructure is not within a normal or acceptable range, an alert should be generated so that the team responsible for the appropriate portion of the database infrastructure can investigate and remedy it.
+   
+Monitoring and measurement for this solution is covered by `Percona Monitoring and Management <https://www.percona.com/software/database-tools/percona-monitoring-and-management>`_. It has a specific dashboard to monitor the Group Replication state and cluster status as a whole. For more information, read `Percona Monitoring and Management Documentation <https://www.percona.com/blog/2021/04/14/percona-distribution-for-mysql-high-availability-with-group-replication-solution/>`_.
```

Comment:
Monitoring and measurement for this solution are

Reply:
fixed